storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -474,7 +475,7 @@ bool Redo_Log_Writer::write_buffer(byte *buf, size_t len) {
 
   if (srv_redo_log_encrypt) {
     IORequest req_type(IORequestLogWrite);
-    fil_space_t *space = fil_space_get(dict_sys_t::s_log_space_first_id);
+    fil_space_t *space = fil_space_get(dict_sys_t::s_log_space_id);
```

Comment:
we are not supposed to use fil_system for redo log. This will give nullptr .
How does this code work?